cab-token-generator/pom.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>com.google.auth</groupId>
+    <artifactId>google-auth-library-parent</artifactId>
+    <version>1.29.1-SNAPSHOT
+    </version><!-- {x-version-update:google-auth-library-parent:current} -->
```

Comment:
```suggestion
    <version>1.30.0</version><!-- {x-version-update:google-auth-library-parent:current} -->
```

This version is going to change as we cut new releases of auth. Last thing before we merge will be to update this to the latest version.

Reply:
I'll leave this comment open as a reminder to update the version before the feature is merged to the main branch.